fluid-project/handbook.floeproject.org
# feat: add audio-tactile cards and new article

## Description

This update adds a new page comprised of 45 recommendation cards for audio-tactile design, as well as a new article introducing this new page.

The designs for the new page and article can be found here: https://www.figma.com/file/jcUv3IqUILArTF8yKdCUK1/Recommendations-page?node-id=0%3A1

## Steps to test

1. Go to the Techniques page
2. Expand the _Authoring Content_ and click on _3D Printed Audio-Tactile Graphics_
3. Click on either "a set of recommendations" or "the 45 recommendations" links to open the cards page
4. Click on "Code Legend" to expand the code legend
5. Click on the "expand all" button
6. Click on the "collapse all" button
7. Click on "Materials and Production"
8. Click on "General"
9. Click on the MPG.3 card

**Expected behavior:** The sections, subsections and cards should all expand and collapse when the expand & collapse all buttons are activated (except those cards without any content to expand: _PG.3_, _PAT.3_, and _CUI.7_). After steps 7 through 9, the _MPG.3_ card should also be expanded. Other cards should expand and collapse likewise, except those cards which do not have an icon since they don't have any additional information

## Additional information

Some interactions or design elements are not implemented exactly as designed for various practical reasons.

## Related issues

Resolves #74
